src/utils/aadGroup.spec.ts

```diff
@@ -90,4 +90,62 @@ describe('utils/aadGroup', () => {
     const actual = await aadGroup.getGroupByDisplayName(validGroupName);
     assert.deepStrictEqual(actual, singleGroupResponse);
   });
+
+  it('correctly get single group id by name using getGroupIdByDisplayName', async () => {
+    sinon.stub(request, 'get').callsFake(async opts => {
+      if (opts.url === `https://graph.microsoft.com/v1.0/groups?$filter=displayName eq '${formatting.encodeQueryParameter(validGroupName)}'?$select=id`) {
+        return {
+          value: [
+            { id: singleGroupResponse.id }
+          ]
+        };
+      }
+
+      return 'Invalid Request';
+    });
+
+    const actual = await aadGroup.getGroupIdByDisplayName(validGroupName);
+    assert.deepStrictEqual(actual, singleGroupResponse.id);
+  });
+
+  it('throws error message when no group was found using getGroupIdByDisplayName', async () => {
+    sinon.stub(request, 'get').callsFake(async opts => {
+      if (opts.url === `https://graph.microsoft.com/v1.0/groups?$filter=displayName eq '${formatting.encodeQueryParameter(validGroupName)}'?$select=id`) {
+        return { value: [] };
+      }
+
+      return 'Invalid Request';
+    });
+
+    try {
+      await aadGroup.getGroupIdByDisplayName(validGroupName);
+      assert.fail('Error expected, but was not thrown.');
```

Comment:
Let's use `assert.rejects` instead of this construction with try/catches just like we do for regular commands.